docs/pages/config-reference.mdx

```diff
@@ -156,6 +156,12 @@ auth_service:
     # certificates and keys (may need to wipe out /var/lib/teleport directory)
     cluster_name: "main"
 
+    # ProxyProtocol enables support for HAProxy proxy protocol when it is turned 'on'.
+    # We recommend it be enabled by an administrator since knowing
+    # whether the service is in front of trusted load balancer is relevant.
```

Comment:
This sentence doesn't make a lot of sense if the default value is `on` IMO as it seems like it's always going to be enabled?

Reply:
Yeah, after a second reading I agree. I think the intent of the original comment here is just to verify with someone knowledgeable. I've tweaked this just a bit:

`Verify whether the service is in front of a trusted load balancer.`